### ISSUE TITLE
Toggle not working

### ISSUE THREAD

**Issue by olegjs** [gregorym/atom-inline-blame#7]
In atom 1.33.0 (Fedora 29), both hotkeys and content menu toggles have no effect.

**Comment by gregorym** [comment]
Is this still an issue?

**Comment by connesy** [comment]
The toggle is not doing anything for me in atom 1.40.1.

**Comment by IllusionVK** [comment]
I also see the same problem @gregorym running Atom 1.34.0 x64 on Mac 10.14.6

**Comment by crstauf** [comment]
Same here; Atom 1.46.0 on macOS 10.15.4.

**Comment by crstauf** [comment]
@gregorym Is there something you're waiting for to fix this? How can we help? The package seems to do nothing at the moment.

**Comment by gregorym** [comment]
I looked at the code and i've actually not implemented that feature :D
I would be great if you could implement it actually. I shouldn't be difficult.

**Comment by crstauf** [comment]
@gregorym I'm sorry, what feature? The blame does not show at all for me, supposedly because the toggle doesn't work...?

**Comment by gregorym** [comment]
@crstauf Which theme are you using?

**Comment by crstauf** [comment]
@gregorym _One Dark_ for both UI and Syntax.

**Comment by gregorym** [comment]
Can you open the dev tools and see if there's any console logs?

**Comment by crstauf** [comment]
@gregorym 

```
Error: Command failed: git blame file/path.php --line-porcelain -L 1,+1
fatal: bad revision '/path/with space/'

    at ChildProcess.exithandler (child_process.js:312)
    at ChildProcess.emit (events.js:194)
    at maybeClose (internal/child_process.js:998)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:265)
```

**Comment by gregorym** [comment]
The file path needs to be wrapper in `" "` to handle path with spaces.

**Comment by crstauf** [comment]
Thanks for the impromptu support, @gregorym, and sorry to have cluttered this issue.

Opened a new issue specific for directories with spaces.

**Comment by gregorym** [comment]
Fixed in version `0.0.10`
